qCC/main.cpp

```diff
@@ -116,6 +116,23 @@ int main(int argc, char **argv)
             commandLine = false;
             lastArgumentIndex += 2;
         }
+		if (argumentsLocal8Bit[lastArgumentIndex].toUpper() == "-VERBOSITY")
```

Comment:
I guess you should check that 'lastArgumentIndex < argumentsLocal8Bit.size()' first?